Makefile

```
@@ -0,0 +1,45 @@
+DIST_VERSION ?= 7
```

The README doesn't mention needing to override DIST_VERSION if building under RHEL8. Can make this automatic by changing to this: 
```suggestion
DIST_VERSION := $(shell sed -r 's/([^:]*:){4}//;s/(.).*/\1/' /etc/system-release-cpe)
```